workflow/sample_files/base-vehicle-ev-charger.xml

```diff
@@ -0,0 +1,689 @@
+                  <DuctInsulationRValue>4.0</DuctInsulationRValue>
+                  <DuctLocation>garage</DuctLocation>
+                  <DuctSurfaceArea>150.0</DuctSurfaceArea>
+                </Ducts>
+                <Ducts>
+                  <SystemIdentifier id='Ducts2'/>
+                  <DuctType>return</DuctType>
+                  <DuctInsulationRValue>0.0</DuctInsulationRValue>
+                  <DuctLocation>garage</DuctLocation>
+                  <DuctSurfaceArea>50.0</DuctSurfaceArea>
+                </Ducts>
+              </AirDistribution>
+            </DistributionSystemType>
+          </HVACDistribution>
+        </HVAC>
+        <WaterHeating>
+          <WaterHeatingSystem>
+            <SystemIdentifier id='WaterHeatingSystem1'/>
+            <FuelType>electricity</FuelType>
+            <WaterHeaterType>storage water heater</WaterHeaterType>
+            <Location>garage</Location>
+            <TankVolume>40.0</TankVolume>
+            <FractionDHWLoadServed>1.0</FractionDHWLoadServed>
+            <HeatingCapacity>18767.0</HeatingCapacity>
+            <EnergyFactor>0.95</EnergyFactor>
+            <HotWaterTemperature>125.0</HotWaterTemperature>
+          </WaterHeatingSystem>
+          <HotWaterDistribution>
+            <SystemIdentifier id='HotWaterDistribution1'/>
+            <SystemType>
+              <Standard>
+                <PipingLength>50.0</PipingLength>
+              </Standard>
+            </SystemType>
+            <PipeInsulation>
+              <PipeRValue>0.0</PipeRValue>
+            </PipeInsulation>
+          </HotWaterDistribution>
+          <WaterFixture>
+            <SystemIdentifier id='WaterFixture1'/>
+            <WaterFixtureType>shower head</WaterFixtureType>
+            <LowFlow>true</LowFlow>
+          </WaterFixture>
+          <WaterFixture>
+            <SystemIdentifier id='WaterFixture2'/>
+            <WaterFixtureType>faucet</WaterFixtureType>
+            <LowFlow>false</LowFlow>
+          </WaterFixture>
+        </WaterHeating>
+        <Vehicles>
+          <Vehicle>
+            <SystemIdentifier id='ElectricVehicle1'/>
+            <VehicleType>
+              <BatteryElectricVehicle>
+                <Battery>
+                  <NominalCapacity>
+                    <Units>kWh</Units>
+                    <Value>100.0</Value>
+                  </NominalCapacity>
+                  <UsableCapacity>
+                    <Units>kWh</Units>
+                    <Value>80.0</Value>
+                  </UsableCapacity>
+                </Battery>
+                <ConnectedCharger idref='EVCharger1'/>
+              </BatteryElectricVehicle>
+            </VehicleType>
+            <MilesDrivenPerYear>10000.0</MilesDrivenPerYear>
+            <HoursDrivenPerWeek>14.0</HoursDrivenPerWeek>
+            <FuelEconomyCombined>
+              <Units>kWh/mile</Units>
+              <Value>0.25</Value>
+            </FuelEconomyCombined>
```

Comment:
I think we need more testing in `test_validation.rb`. I tried changing the units to "mpg" and it ran successfully. I looked at the `in.xml` file and it didn't add a default kWh/mile value, so I guess it just interpreted my mpg value as kWh/mile?

Reply:
Turned out to be an error in the schematron, I corrected and added a new test here 60bc114b178f4fb95a7f7ca8c3fb76aba046685f